src/next-app/pages/quote-request.tsx

```diff
@@ -0,0 +1,328 @@
+import type { NextPage } from 'next';
+import { ChangeEvent, useState } from 'react';
+import Button from '../components/Button';
+import TextField from '../components/TextField';
+import Header from '../components/Header';
+import Link from '../components/Link';
+import Modal from '../components/Modal';
+import styles from '../styles/pages/QuoteRequest.module.css';
+import DropdownField from '../components/DropdownField';
+import TextMultiField from '../components/TextMultiField';
+import SingleDropdownField from '../components/SingleDropdownField';
+import carData from '../data/data.json';
+import validateEmail from '../utils/validateEmail';
+import TextArea from '../components/TextArea';
+import ShopCard from '../components/ShopCard';
+// import { ImagePicker } from 'react-file-picker';
+
+interface carModels {
+    [make: string]: string[],
+}
+
+interface checkedShops {
+    [shop: string]: boolean
+}
+
+const QuoteRequestPage: NextPage = () => {
+    const [name, setName] = useState('');
+    const [make, setMake] = useState('');
+    const [customMake, setCustomMake] = useState('');
+    const [model, setModel] = useState('');
+    const [modelYear, setModelYear] = useState('');
+    const [customModel, setCustomModel] = useState('');
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [phoneNumber, setPhoneNumber] = useState('');
+    const [email, setEmail] = useState('');
+    const [emailErrors, setEmailErrors] = useState<Set<string> | undefined>(undefined);
+    const [preferredContact, setPreferredContact] = useState('');
+    const [notes, setNotes] = useState('');
+    const [partPreferenceSeller, setPartPreferenceSeller] = useState('');
+    const [partPreferenceType, setPartPreferenceType] = useState('');
+    const [startDate, setStartDate] = useState('');
+    const [endDate, setEndDate] = useState('');
+    const [open, setOpen] = useState(false);
+    const [checkedShops, setCheckedShops] = useState({} as checkedShops);
+    const [disableSubmitShops, setDisableSubmitShops] = useState(true);
+    const [checkedShopsDisplay, setCheckedShopsDisplay] = useState([] as JSX.Element[]);
+
+    let makes: string[] = [];
+    let models: carModels = {};
+
+    let valid = false;

```

Comment:
This variable is set to `false` and never changed. Shouldn't you be setting it to `true` somewhere in the code to indicate the form inputs have become valid?